### ISSUE TITLE
Finish-eGon100RE-gas-grid-fails

### ISSUE THREAD

**Issue by CarlosEpia** [openego/powerd-data#288]
The task gas_grid.insert-gas-data is failing when scenario is eGon100RE.  Can only be merged after https://github.com/openego/powerd-data/pull/122

**Comment by CarlosEpia** [comment]
Since #122 is still work in progress, once it is merged into dev, a new branch should be created and this two commits should be cherry-picked:
5c47f3c5670fc80b66b5ae6506b1cc731b922021
21dbe1dd5b5d04eabdd8367ad5f612f1b8811011
a3a4352ac08be3462a50f895eeb162442292e6da
d298e3d4317eb07ee8ecb406ddca54e8445fc136
c9a2360711f385a474dc499d0d97fc292030591c
0785576227626ca680afe6372da4e6016e4370fd
